Assets/Scripts/Event Manager.cs

```diff
@@ -0,0 +1,53 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+
+[CreateAssetMenu(fileName = "EventManager", menuName = "Scriptable Objects/EventManager")]
+public class EventManager : ScriptableObject
+
+{
+    public static EventManager instance;
+
+    public delegate void ButtonClickEvents();
```

Comment:
A delegate is an object. It should be named like how a class is named.

In this case I could call the delegate "ButtonClickEvent" instead of "ButtonClickEvents", because we are creating the blueprint for a single instance of an event, not a list of events or something of that nature.

Reply:
Resolved